etc/adminhtml/system.xml

```diff
@@ -246,6 +246,16 @@
                     <label>Send shipping details</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                 </field>
+                <field id="threshold_amount" type="text" sortOrder="32" translate="label" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
Could we move this setting to the advanced section, just to make it a little harder for the customer to do, since we don't really want them hiding the button at all

Reply:
We don't have advanced section in m2 version